src/python/turicreate/data_structures/sframe.py

```diff
@@ -2373,7 +2373,7 @@ def head(self, n=10):
         """
         return SFrame(_proxy=self.__proxy__.head(n))
 
-    def to_dataframe(self):
+    def to_dataframe(self, explore=False):
```

Comment:
Let's not make explore be a parameter on another method. It should stay as its own method.

Reply:
Actually, if we can get the contents of the image into the dataframe, we can just do that all the time (and not add a new parameter here).